lib/components/viewers/styled.ts

```diff
@@ -145,8 +148,10 @@ export const PatternRowItem = styled.li<{ roundedTop?: boolean }>`
   }
 
   &:last-of-type .header.stop-view {
```

Comment:
Can we not take advantage of selectors instead of adding a new prop? Would make a lot of this code much much cleaner